tests/js/dictionary-tests.js

```diff
@@ -35,7 +35,7 @@ module.exports = {
         let realm = new Realm({schema: [DictSchema]})
         realm.write(()=> realm.create(DictSchema.name, { a: {x:1, y:2, z:"hey"} } ))
         let data = realm.objects(DictSchema.name)[0]
-
+        TestCase.assertTrue(data.a instanceof(Realm.Dictionary), "Should be a dictionary");
```

Comment:
```suggestion
        TestCase.assertTrue(data.a instanceof Realm.Dictionary, "Should be a dictionary");
```

Treating a binary operator like a function looks odd

Reply:
Also, maybe use `TestCase.assertInstanceOf`?